### PR TITLE
Rewrote the routes

### PR DESCRIPTION
**Pull request by Racenna**

### CODE REVIEW

**Comment by Racenna** [comment]
Removed all withRouter and use hook useParams in ProfileContainer.jsx

File: src/App.js [hunk 1]
```diff
@@ -40,29 +44,42 @@ const App = () => {
...
-        </div>
-      </div>
+      <AuthContext.Provider value={isAuth}>
+        <Router>
+          <div className='app-wrapper'>
+            <Header />
+            <Navbar />
+            <div className='app-wrapper-content'>
+              <Route path='/' exact render={() => <Redirect to='/profile' />} />
+              {/* <Route
```
Comment (worksnk) [inline src/App.js:54]: Remove commented code

File: src/App.js [hunk 2]
```diff
@@ -14,13 +14,17 @@ import UsersContainer from './components/Users/UsersContainer';
 import ProfileContainer from './components/Profile/ProfileContainer';
 import Header from './components/Header/Header';
 import Preloader from './components/common/Preloader/Preloader';
+import PrivateRoute from './components/common/PrivateRoute/PrivateRoute';
 // Style
 import './App.css';
 // React.lazy
 const Login = React.lazy(() => import('./components/Login/Login'));
 
+export const AuthContext = createContext();
```
Comment (worksnk) [inline src/App.js:23]: why do you need this context?

File: src/App.js [hunk 3]
```diff
@@ -14,13 +14,17 @@ import UsersContainer from './components/Users/UsersContainer';
 import ProfileContainer from './components/Profile/ProfileContainer';
 import Header from './components/Header/Header';
 import Preloader from './components/common/Preloader/Preloader';
+import PrivateRoute from './components/common/PrivateRoute/PrivateRoute';
 // Style
 import './App.css';
 // React.lazy
```
Comment (worksnk) [inline src/App.js:20]: Remove this comments. They are not really useful. It's clear without them what is what

**Review by worksnk — CHANGES_REQUESTED** [review]

**Review by worksnk — APPROVED** [review]

**Pull request merged**